source/Mission.cpp

```diff
@@ -1144,6 +1147,9 @@ void Mission::Do(const ShipEvent &event, PlayerInfo &player, UI *ui)
 		}
 	}
 
+	if(event.Type() & ShipEvent::DISABLE && event.Target().get() == player.Flagship())
```

Comment:
```suggestion
	if((event.Type() & ShipEvent::DISABLE) && event.Target().get() == player.Flagship())
```

Reply:
Isn't the bitwise AND evaluated before the logical AND?

Reply:
Yeah but that's the common advice when using bitwise operations, because the order of evaluation for them is weird. It's basically done for clarity.